artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java

```diff
@@ -110,25 +112,27 @@ public Object execute(ActionContext context) throws Exception {
          return null;
       }
 
+      int printingMaxRows = this.maxRows != null ? this.maxRows : DEFAULT_MAX_ROWS;
+
       if (verbose) {
          context.out.println("filter is '" + filter + "'");
-         context.out.println("maxRows='" + maxRows + "'");
+         context.out.println("maxRows='" + printingMaxRows + "'");
```

Comment:
why the need for printingMaxRows....

Reply:
I only change the variable name. The maxRows was printed using verbose before my changes.

Reply:
i was meaning no need for the printingMaxRows vairable, just keep it all back on single maxRows.